src/java/org/apache/cassandra/net/InboundConnectionInitiator.java

```diff
@@ -69,6 +71,8 @@ public class InboundConnectionInitiator
 {
     private static final Logger logger = LoggerFactory.getLogger(InboundConnectionInitiator.class);
 
+    private static final NoSpamLogger noSpam5m = NoSpamLogger.getLogger(logger, 5, TimeUnit.MINUTES);
```

Comment:
I am not completely sure how long these security scans are in general. I feel like 1 minute might not be enough. Does not hurt to have 5.

Reply:
think this is scan dependent so hard to say

Reply:
The security scan run about 6-10 minutes and generate 130 stack traces.